test.ts

```diff
@@ -699,8 +707,12 @@ Deno.test("get columns from finalized query throws", function () {
 Deno.test("date time is correct", function () {
   const db = new DB();
   // the date/ time is passed from JS and should be current (note that it is GMT)
-  const [[now]] = [...db.query("SELECT current_timestamp")];
-  assertEquals(new Date(now + "Z"), new Date());
+  const [[now]] = [...db.query("SELECT STRFTIME('%Y-%m-%d %H:%M:%f', 'NOW')")];
+  const jsTime = new Date().getTime();
+  const dbTime = new Date(`${now}Z`).getTime();
+  // to account for runtime latency, up to one second difference is ok
```

Comment:
```suggestion
  // to account for runtime latency, a small difference is ok
```

Reply:
@dyedgreen Looks like I forgot to update the comment in the last change: nice catch; thanks!